meal_planner/ui/list_recipes.py

```
@@ -7,7 +11,22 @@
 
 
 def format_recipe_list(recipes_data: list[dict]) -> FT:
-    """Formats the recipe list data into a list of recipe items."""
+    """Format a list of recipes as an interactive HTML list.
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)